### PR TITLE
Fix usage of temporary dir (/tmp/) [for the remaining]

### PR DESCRIPTION
**Pull request by joaoantoniocardoso**

### CODE REVIEW

**Comment by JPTIZ** [comment]
I really would appreciate if you could be more specific in PR names :) Prefixing with `lib: ` could be a good start.

**Comment by JPTIZ** [comment]
Oh I see you just changed the name to clarify

File: src/examples/evas/evas-images3.c [hunk 1]
```diff
@@ -30,10 +30,10 @@ static const char *commands = \
   "commands are:\n"
   "\tw - write new pixel data to image\n"
   "\ti - print image info\n"
-  "\ta - save noise image to disk (/tmp dir)\n"
+  "\ta - save noise image to disk (TEMP dir)\n"
   "\th - print help\n";
 
-const char *file_path = "/tmp/evas-images2-example.png";
+const char *filename = "evas-images2-example.png";
```
Comment (Coquinho) [inline src/examples/evas/evas-images3.c:36]: This one is kind strange. It don't seems to be used anywhere.

Comment (joaoantoniocardoso) [reply src/examples/evas/evas-images3.c:36]: Yeah, indeed it isn't used anywhere, but as it is a copy of the same template of its brothers `evas-image[2345].c` I changed here too.

**Review by JPTIZ — APPROVED** [review]

**Pull request merged**